Libraries/LibGfx/ImageFormats/TinyVGLoader.cpp

```diff
@@ -471,25 +471,27 @@ ErrorOr<NonnullRefPtr<TinyVGDecodedImageData>> TinyVGDecodedImageData::decode(St
     return TRY(adopt_nonnull_ref_or_enomem(new (nothrow) TinyVGDecodedImageData({ header.width, header.height }, move(draw_commands))));
 }
 
-void TinyVGDecodedImageData::draw(Painter& painter) const
+void TinyVGDecodedImageData::draw_transformed(Painter& painter, AffineTransform transform) const
 {
+    // FIXME: Correctly handle non-uniform scales.
```

Comment:
Why would you revert that?

Reply:
I'm did an error while rebasing, I fixed it